### PR TITLE
#5326 Fix the criteria definition

### PR DESCRIPTION
**Pull request by jagpreetstamber**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by jagpreetstamber** [comment]
Fixes issue https://github.com/Azure/azure-rest-api-specs/issues/5326

File: specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json [hunk 1]
```diff
@@ -728,8 +728,11 @@
       ],
       "properties": {
         "criteria": {
-          "$ref": "#/definitions/Criteria",
-          "description": "Severity of the alert"
+          "type": "array",
```
Comment (praries880) [inline specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json]: Is the type "LogToMetricAction" sent in by the user or is it something that gets returned form the server side and the user treats as a read only type?

This seems like a breaking change.. has the SDK without these changes already been released?

Comment (jagpreetstamber) [reply specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json]: LogToMetricAction is sent by user in the request payload, The criteria filed was always supposed to be arrray but sdk got released with incorrect definition.

https://github.com/Azure/azure-rest-api-specs/issues/5326 has been opened for same.

**Review by KrisBash — APPROVED** [review]
Given my understanding that this is aligning client to service behavior, lgtm

**Pull request merged**